assets/blocks/quiz/quiz-block/quiz-edit.js

```diff
@@ -65,7 +81,9 @@ const QuizEdit = ( props ) => {
 	);
 	const pagination = props?.attributes?.options?.pagination;
 	const showPaginationProgressBar =
-		pagination?.paginationNumber && pagination?.showProgressBar;
+		pagination?.paginationNumber &&
+		pagination?.showProgressBar &&
+		pagination.paginationNumber < questionCount;
```

Comment:
Should we use `?` here?
```
pagination?.paginationNumber < questionCount;
```

Reply:
> Should we use `?` here?
> 
> ```
> pagination?.paginationNumber < questionCount;
> ```

Good point! I think as it checks from the left, if the first one (pagination?.paginationNumber) returns false, it won't go to the rest of the && checks, it should short circuit at the beginning. It should be alright, but maybe adding another "?" check can be good for consistency and also for just in case situations. Thanks!